components/storage-sync/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "storage-sync"
+edition = "2018"
+version = "0.1.0"
+authors = ["application-services@mozilla.com"]
+license = "MPL-2.0"
+
+[dependencies]
+error-support = { path = "../support/error" }
+failure = "0.1.6"
+ffi-support = { path = "../support/ffi" }
+interrupt = { path = "../support/interrupt" }
+lazy_static = "1.4.0"
+log = "0.4"
+serde = "1.0.104"
+serde_json = "1.0.44"
+sql-support = { path = "../support/sql" }
+sync15 = { path = "../sync15" }
```

Comment:
Can't use sync15 directly if we're going to land in desktop easily. Use sync15-traits.